versions.cfg

```diff
@@ -85,3 +85,10 @@ zc.recipe.testrunner = 2.2
 zest.releaser = 6.22.2
 zipp = 3.7.0
 zodbupdate = 1.5
+
+[versions:python36]
```

Comment:
This is not enough for Python 3.7.

I tried it with `venv`, `virtualenv`, `buildout==2.13.5` as well as `buildout==3.0.0rc2` in any case I get:

```
Installing scripts.
While:
  Installing scripts.
  Getting distribution for 'typing-extensions>=3.6.4'.
Error: Picked: typing-extensions = 4.0.1
```

I wanted to mitigate this problem by using the `[versions:python3.7]` section.

This is not a problem for the `tox` run as in `tox.ini` we just install the `tests` part.

Buildout 2 is missing this commit https://github.com/buildout/buildout/commit/db3d6e2fbf5d7ff2cc4b2507253c7a221cfc3e32  to support `[versions:python3.7]`.

I am going to try to cherry pick it to the 2.x branch.

Reply:
See https://github.com/buildout/buildout/pull/605 as the PR on buildout 2.x.